chart/test/k3d-ci.yaml

```diff
@@ -0,0 +1,6 @@
+# k3s overrides for ci
+# --------------------
+
+# don't pull use, existing images
```

Comment:
Should this actually just be in the base test.yaml? The microk8s also uses existing images, so should never try to pull..